recipes/freeglut/all/conanfile.py

```diff
@@ -24,33 +25,79 @@ class freeglutConan(ConanFile):
         "print_errors_at_runtime": [True, False],
         "print_warnings_at_runtime": [True, False],
         "replace_glut": [True, False],
+        "with_wayland": [True, False],
+
     }
     default_options = {
         "shared": False,
         "fPIC": True,
-        "gles": False,
+        "gles": True,
```

Comment:
What's the default value in the upstream? Why can not we keep False?

Reply:
The default is enabled for Android and disabled otherwise. See [here](https://github.com/FreeGLUTProject/freeglut/blob/c0e9657c1e44ce4ae7091f317215fa2c48541643/CMakeLists.txt#L63-L68). I've updated it to do likewise. I'd figured that enabling GLES support by default would reduce the likelihood of needing to rebuild the binaries for cases that required it.